### DISCUSSION TITLE
GraphQL to get the current pending payout?

### DISCUSSION THREAD

**Discussion by Sh4d** [community/community#133445] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to pull the current pending balance for payout that's in our stripe connect account, but I can't seem to find anything that would let me accomplish that. Is this possible?

I saw https://github.com/orgs/community/discussions/77190#discussioncomment-8247804 and was hopeful I could just add them all up, but when I run that query all the nodes are null.

**Comment by AntA001** [comment]
I do not think there is native support for what you are asking,

A work arround of the top of my head would be to implement some sort of resolver that calls the Stripe REST API endpoints to fetch the balance information.

**Reply by Sh4d** [reply, marked as answer]
Unfortunately stripe connect doesn't seem to offer an api as far as I can tell.

I contacted Github support and they confirmed retrieving the pending balance isn't possible right now.

**Comment by apksremi** [comment]
It looks like you're running into trouble retrieving the current pending balance for payouts in your Stripe Connect account. Based on the link you shared, it seems like the approach you're trying might not be working as expected due to null values in the query results.

Here's a step-by-step method you can follow to pull the current pending [balance](https://reminiapks.in/):

1. **Retrieve Account Information**: Make sure you’re fetching the correct account details. Use the Stripe API to get your account's balance information:
   
   ```bash
   curl https://api.stripe.com/v1/balance \
     -u sk_test_your_secret_key:
   ```

   This will give you a JSON response with the balance information, including pending and available amounts.

2. **Filter Pending Balance**: Look specifically for the `pending` field in the response. It should show the amount currently pending in your Stripe account.

3. **Check Payouts API**: If you need more detailed payout information, use the Payouts API:
   
   ```bash
   curl https://api.stripe.com/v1/payouts \
     -u sk_test_your_secret_key:
   ```

   This will list all payouts and their statuses. You can then aggregate the data as needed.

4. **Consult Stripe Support**: If you're still having issues, it might be best to reach out to Stripe's support directly or consult their [official documentation]

If you need any further assistance with the API calls or understanding the response data, feel free to ask!

**Reply by Sh4d** [reply]
What is this chatgpt shit. There's no stripe connect api. Don't spam Github with your wrong answers.
